schema.sql

```diff
@@ -41,4 +41,29 @@ ALTER TABLE animals ADD owner_id INT;
 
 ALTER TABLE animals ADD CONSTRAINT c2
       FOREIGN KEY(owner_id) 
-	  REFERENCES owners(id);
\ No newline at end of file
+	  REFERENCES owners(id);
+
+CREATE TABLE vets (
+    id SERIAL,
+    name VARCHAR(40),
+    age INT,
+    date_of_graduation DATE,
+    PRIMARY KEY(id)
+);
+
+Create table specialities (
```

Comment:
- [OPTIONAL] The name of this table should be `specializations` as per the project [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_join_table.md), please kindly update the table name.